### ISSUE TITLE
Customize Security Guard Membership Problem 

### ISSUE THREAD

**Issue by ghost** [kahanu/Security-Guard#19]
Hello Everyone,

I use Security Guard Membership for MVC3

The scenario is i want to create a CMS for administrator, when admin go to url e.g: site.com/admin  . It will go to dashboard as default if admin logined otherwise URL will go to login page for admin to login. And i also want to create a login page for user who want to buy some thing from my site.
1. How can i create 2 login page for admin and for user ? when admin go to some URL in CMS, The site will go to login page for admin. And if the client want to do some thing like order goods ... they are required to login and the site will go to login page for them. Im so confused and  get stuck in this.

2.Can i use implementing method create user of membership to add user for the client ?

Please give me suggestions, thanks in advance !

**Comment by kahanu** [comment]
@NevenHuynh - yes I think you are a bit confused, sorry about that.  

SecurityGuard is your site for Administrators to manage membership for your application.  That's all.  It's entirely up to you on how to build your application to do what you want, just like any other application.

Think of SecurityGuard as a plugin into your site that allows you to manage your membership.

When you assign users to Roles in your application, you need to determine what permissions those Roles have in your application.  Customers of your application would simply have access to purchase products.  They would not have access to SecurityGuard or any other Administrator areas.

Regarding logins, you don't need to do anything special to make this work.  Everyone uses the same login form, and the application determines what the user has access to after they are authenticated.

Regarding creating users, you can create users manually in the SecurityGuard Dashboard, or you can have your customers use the Register page to register themselves.  Anyone who registers this way will not have SecurityGuard permissions, so you don't have to worry about them gaining access to your membership system.

Bottom line is, SecurityGuard did not change anything in the way you would normally build your applications.  It just provided you with a full-fledged User Interface for managing your membership.  Building your application is still up to you.

Also, regarding the site.com/admin Url, this is fine if you want to create an overall Administrator area to manage your application (Products, Customers, etc), but you should keep the location of SecurityGuard unchanged.  It just makes this easier.  Build your web app admin site and then just create a link to the SecurityGuard Dashboard to manage membership.

I hope this answers your question.

Thanks,

King Wilder
